.gitignore

```diff
@@ -51,6 +51,7 @@ _packaging/windows/file_version_info.txt
 _packaging/windows/output
 
 # MacOS
+.DS_Store
```

Comment:
This would probably be better in your own global ignore file, since it isn't specific to Gaphor.

Reply:
Agreed, corrected